packages/template-typescript-minimal/app/pages/home.tsx

```diff
@@ -19,7 +19,6 @@ const style = `
 body {
     background: linear-gradient(-45deg, #e73c7e, #23a6d5, #ee7752);
     background-size: 400% 400%;
-    animation: gradient 10s ease 5;
```

Comment:
Lets really get rid of this. We can re-enable it later. #makes-my-computer-slooow